controllers/telemetry/logpipeline_controller_test.go

```diff
@@ -405,3 +515,20 @@ func validateLoggingOwnerReferences(ownerReferences []metav1.OwnerReference) err
 
 	return nil
 }
+
+func validateKeyExistsInFluentbitSectionsConf(ctx context.Context, key string) bool {
+	configMapLookupKey := types.NamespacedName{
+		Name:      testLogPipelineConfig.SectionsConfigMap.Name,
+		Namespace: testLogPipelineConfig.SectionsConfigMap.Namespace,
+	}
+	var fluentBitCm corev1.ConfigMap
+	err := k8sClient.Get(ctx, configMapLookupKey, &fluentBitCm)
+	if err != nil {
+		fmt.Printf("could not get configmap: %s: %s", configMapLookupKey.Name, err.Error())
```

Comment:
Maybe rather make the return type of this function `error` and return this as an error. That would make it more obvious on failure and does not require to search in logs.